docs/source/classification/fbeta_score.rst

```diff
@@ -1,22 +1,64 @@
 .. customcarditem::
-   :header: FBeta Score
+   :header: F-Beta Score
    :image: https://pl-flash-data.s3.amazonaws.com/assets/thumbnails/tabular_classification.svg
    :tags: Classification
 
 .. include:: ../links.rst
 
-###########
-FBeta Score
-###########
+############
+F-Beta Score
+############
 
 Module Interface
 ________________
 
+FBetaScore
+^^^^^^^^^^
+
 .. autoclass:: torchmetrics.FBetaScore
     :noindex:
 
+BinaryFBetaScore
+^^^^^^^^^^^^^^^^
+
+.. autoclass:: torchmetrics.BinaryFBetaScore
+    :noindex:
+
+MulticlassFBetaScore
+^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: torchmetrics.MulticlassFBetaScore
+    :noindex:
+
+MultilabelFBetaScore
+^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: torchmetrics.MultilabelFBetaScore
+    :noindex:
+
 Functional Interface
 ____________________
 
+fbeta_score
+^^^^^^^^^^^
+
 .. autofunction:: torchmetrics.functional.fbeta_score
     :noindex:
+
+binary_fbeta_score
+^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: torchmetrics.functional.binary_fbeta_score
+    :noindex:
+
+multiclass_fbeta_score
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: torchmetrics.functional.multiclass_fbeta_score
+    :noindex:
+
+multilabel_fbeta_score
```

Comment:
Isn't it the same as `multiclass_fbeta_score`, just that the label sum per sample is exactly one?